content/basics/ably.textile

```diff
@@ -0,0 +1,86 @@
+
+* Live Chat - chat for customer support or community engagement
+* Data Broadcast - broadcast realtime event data to millions of devices
+* Data Sync - keep backend and frontend clients in realtime sync
+* Multiplayer Collaboration - collaborative workspaces for synchronous teamwork
+* Notifications - deliver in-app notifications in realtime to users
+
+h2(#about). Ably features
+
+Ably provides a number of features that enhance realtime experiences, some of the key Ably features are: 
+
+* "Pub/sub with channels":#pubsub
+* "Presence and occupancy":#presence-and-occupancy
+* "Message storage and history":#message-storage-and-history
+* "Push Notifications":#push-notifications
+
+h3(#pubsub). Pub/sub with channels
+
+Ably uses the pub/sub architectural pattern to enable any number of publishers to publish data and any number of subscribers to subscribe to that data. This works with Ably's "concept of channels":/channels, enabling publishers to publish to a single location and subscribers to subscribe to a single source.
+
+h3(#presence-and-occupancy). Presence and occupancy
+
+Ably’s "presence":/presence-occupancy/presence feature enables clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events, such as a client entering or leaving the presence set. Each member in the presence set can have data associated with them. You can also request a list of clients or devices that are connected to a channel at a particular point in time via the REST API.
+
+"Occupancy":/presence-occupancy/occupancy provides metrics about the clients attached to a channel, including a count of the number of clients attached, the number of presence members, and a breakdown of clients by authorization.
+
+h3(#message-storage-and-history). Message storage and history
+
+By default messages are only stored for two minutes on Ably’s servers. Messages are retrieved automatically by the SDK in the event of a lost connection within these two minutes. However, if you've enabled message persistence and the disconnection is longer than two minutes you can use the history feature to retrieve messages.
+
+Ably's "message storage and history":/storage-history enables messages to be accessed after initially being sent. When enabled you can retrieve the last message sent to a channel using rewind, or enable persistence of all messages for 24 hours for the free account tier or 72 hours for paid account tiers. These messages can be retrieved using history. You can also persist messages to external storage services such as AWS through the use of "Ably Integrations":/integrations.
+
+h3(#push-notifications). Push notifications
+
+Ably's "Push Notifications":/push feature enables messages to be pushed to subscribers without needing a device to maintain a connection to Ably.
```

Comment:
"enables messages to be pushed to subscribers on mobile devices without needing to maintain a connection to Ably.